packages/core/src/browser/theming.ts

```diff
@@ -134,7 +131,8 @@ export class ThemeService {
      * The default theme. If that is not applicable, returns with the fallback theme.
      */
     get defaultTheme(): Theme {
-        return this.themes[this._defaultTheme || this.fallbackTheme] || this.themes[this.fallbackTheme];
+        const defaultTheme = FrontendApplicationConfigProvider.get().defaultTheme;
+        return defaultTheme && this.themes[defaultTheme] || this.themes['dark'];
```

Comment:
Once you have moved the defaults theme and icon to the DEFAULT application props, it should be fine to call `this.themes[Application.DEFAULT.frontend.config.defaultTheme]`. No `'dark'` as a string.

Reply:
It cannot here. `dark` is not a default value in this context, but a fallback value if for some reason the default theme was not loaded.

Reply:
ah, I got what you mean now!